src/views/shop/registrationShopping.vue

```diff
@@ -168,6 +169,9 @@ export default {
 				item => item.id_product === idProduct
 			);
 			this.$store.commit('shoppingCar/REMOVE', indexInStore);
+		},
```

Comment:
This method is never declared but is invoked

Reply:
I do not know, i didn't change the original code of this lines

Reply:
It is possible